packages/tools/devtools/devtools-example/package.json

```diff
@@ -26,14 +26,18 @@
 		"prettier": "prettier --check . --ignore-path ../../../../.prettierignore",
 		"prettier:fix": "prettier --write . --ignore-path ../../../../.prettierignore",
 		"rebuild": "npm run clean && npm run build",
-		"start": "pm2 start pm2.ecosystem.config.js",
+		"start": "concurrently \"npm:watch:*\" \"npm:start:client\"",
 		"start:client": "webpack serve --config webpack.config.js",
 		"stop": "pm2 stop pm2.ecosystem.config.js",
```

Comment:
Just to confirm, do we want to  keep remaining pm2 script related stuff around?
There is readme documentation and the entire pm2.ecosystem.config.js file

Reply:
Good catch, we should remove the `stop` script and the references in the docs/readme. @chentong7 can you do that when you have a chance?

Reply:
Gotcha